sql/create-db.sql

```diff
@@ -0,0 +1,76 @@
+-- Set up a PostgreSQL Database for use as the backing
```

Comment:
the triggers are basically to implement a TTL system for EC2 Ids, since they are not universally unique and can be reused.  Basically, I want to have something that says anything older than 1 week is absolutely, 100% unexpected.

I don't have strong thoughts on triggers, but am also not a postgres expert.  This could be a bad idea for sure!